Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            PersonInfoAskerAndPrinter();
+            PersonInfoAskerAndPrinter();
+        }
+        private static void PersonInfoAskerAndPrinter()
+        {
+            string firstName = StringPrompter("What is your first name: ");
+            string lastName = StringPrompter("What is your last name: ");
+            int age = IntPrompter("What is your age: ");
+            float weight = FloatPrompter("what is your weight in Kg: ");
+            float height = FloatPrompter("what is your height in cm: ");
+            Console.WriteLine($"{firstName} {lastName} is {age} years old, his weight is {weight} kg and his height is {height} cm. ");
+            float bmi = BMICalculator(weight, height / 100);
+            Console.WriteLine($"Your BMI is {bmi}");
+        }
+        public static string StringPrompter(string message)
+        {
+            Console.Write(message);
+            return Console.ReadLine();
+        }
+        public static int IntPrompter(string message)
+        {
+            Console.Write(message);
+            return int.Parse(Console.ReadLine());
+        }
+        public static float FloatPrompter(string message)
+        {
+            Console.Write(message);
+            return float.Parse(Console.ReadLine());
+        }
+        public static float BMICalculator(float weight, float height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'BMICalculator' to match pascal case naming rules, consider using 'BmiCalculator'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4864051)